node/internal/errors.ts

```diff
@@ -164,6 +163,17 @@ export const uvExceptionWithHostPort = hideStackFrames(
   },
 );
 
+/** Copied from `../util.ts` to avoid a circular dependency. */
```

Comment:
I don't think we should have multiple copies like this since the copies can get out of sync.

Reply:
If imported, a circular dependency is introduced. What are some alternatives?

Reply:
I think you should be able to import [this file](https://github.com/denoland/deno_std/blob/main/node/internal/error_codes.ts) to get the error code this PR adds to `internal/util.mjs`.

Reply:
Ok, I avoided using `getSystemErrorName` altogether as it's deprecated and instead used `uvErrmapGet`, as recommended [here](https://github.com/nodejs/node/blob/7e0097d8a33fa7adbc1f298cbf647f6d2fd403e8/lib/internal/errors.js#L628-L633). Funnily enough, even Node hasn't implemented this improvement yet.

Reply:
So the initial issue was that importing `getSystemErrorName` from `internal/util.mjs` introduced the circular dependency. I've made the above change and now `parallel\test-dgram-recv-error.js` is failing on Windows because it uses an error number of `-1`, which isn't caught by `uvErrmapGet`, but was caught by `getSystemErrorName`.

I'm not sure what can be done. What do you think?

Reply:
Ok, I've reverted my previous change. Now, `getSystemErrorName` has been moved from `node/util.ts` to `node/_utils.ts`. This removes the circular dependency and retains proper error handling behaviour in Windows.